cmd/dashboard.go

```diff
@@ -84,7 +87,7 @@ dapr dashboard -k -p 9999
 			os.Exit(1)
 		}
 
-		if dashboardLocalPort <= 0 {
+		if dashboardLocalPort < 0 {
```

Comment:
How does this work for standalone mode? Standalone mode calls the dashboard CLI directly and passes a zero value to this code:
https://github.com/dapr/dashboard/blob/febc4a7bca4cb992bc3fdd9799a0a87b5922f8f6/cmd/webserver.go#L72

Have you tried running it locally?

Reply:
@imneov

Reply:
The standalone mode is different. I think there are two main reasons:
1. The dashboard is already running in the K8S environment, so `dashboardLocalPort==0` will not generate a new process. But in standalone mode `dashboardLocalPort==0` will generate a new process,
2. The dashboard already running in standalone mode has no suitable method to obtain the listening port information

Reply:
When `dashboardLocalPort` is 0 for standalone mode, it passes the port number as is and ends up calling [this](https://github.com/dapr/dashboard/blob/febc4a7bca4cb992bc3fdd9799a0a87b5922f8f6/cmd/webserver.go#L108-L115):

```go
	srv := &http.Server{
		Handler:      r,
		Addr:         fmt.Sprintf("0.0.0.0:%v", port),
		WriteTimeout: 15 * time.Second,
		ReadTimeout:  15 * time.Second,
	}
	fmt.Printf("Dapr Dashboard running on http://localhost:%v\n", port)
	log.Fatal(srv.ListenAndServe())
```
Port 0 should identify the next available free port and start the server. However, the user won't know which port is it running on. The `fmt.Printf` will still print `localhost:0`. We will need to fix this. Can you please create an issue to track the same?

We will need to use `net.Listen` like in https://stackoverflow.com/a/43425461/4654175

cc @mukundansundar

Reply:
OK, I will try to add it

Reply:
@shubham1172 
I commit changes on a pull request  https://github.com/dapr/dashboard/pull/205/files